docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -1133,150 +1130,169 @@
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
 
-      <itemizedlist>
-        <listitem>
-          <para>Press the <emphasis role="bold">Generate</emphasis>
-          button.</para>
+        <para>To generate a list:</para>
 
-          <informaltable colsep="1" frame="all" rowsep="1">
-            <?dbfo keep-together="always"?>
+        <itemizedlist>
+          <listitem>
+            <para>Press the <emphasis role="bold">Generate</emphasis>
+            button.</para>
 
-            <tgroup cols="2">
-              <colspec colwidth="49.50pt" />
+            <informaltable colsep="1" frame="all" rowsep="1">
+              <?dbfo keep-together="always"?>
 
-              <colspec />
+              <tgroup cols="2">
+                <colspec colwidth="49.50pt" />
 
-              <tbody>
-                <row>
-                  <entry><inlinegraphic fileref="images/tip.jpg" /></entry>
+                <colspec />
 
-                  <entry><para>Sasha Server typically runs Xref at the times
-                  scheduled when deployed.</para><para> </para></entry>
-                </row>
-              </tbody>
-            </tgroup>
-          </informaltable>
-        </listitem>
-      </itemizedlist>
+                <tbody>
+                  <row>
+                    <entry><inlinegraphic fileref="images/tip.jpg" /></entry>
 
-      <para>To view results:</para>
+                    <entry><para>Sasha Server typically runs Xref at the times
+                    scheduled when deployed.</para><para> </para></entry>
+                  </row>
+                </tbody>
+              </tgroup>
+            </informaltable>
+          </listitem>
+        </itemizedlist>
 
-      <itemizedlist>
-        <listitem>
-          <para>Click on one of the hyperlinks (Found Files, Orphan Files,
-          Lost Files, Directories, or Errors/Warnings).</para>
-        </listitem>
-      </itemizedlist>
+        <para>To view results:</para>
 
-      <sect2 id="ECLWatch_WorkingWithXref">
-        <title>Working with XREF results:</title>
+        <itemizedlist>
+          <listitem>
+            <para>Click on one of the tabs (Found Files, Orphan Files, Lost
+            Files, Directories, or Errors/Warnings).</para>
+          </listitem>
+        </itemizedlist>
 
-        <para>After XRef completes you will see a list of available reports.
-        Click on one of the links to see the results page of each type.</para>
+        <sect3 id="ECLWatchXREFMultiThor">
+          <title>XREF with multiple Thor clusters</title>
 
-        <figure>
-          <title>XRef Errors</title>
+          <para>XREF runs on the primary Thor cluster. For a configuration
+          with more than one Thor cluster, ECL Watch only displays the Thor
+          nodes listed as the primary Thor (the Thor cluster whose name
+          matches the nodeGroup in the configuration.)</para>
```

Comment:
It doesn't display nodes anywhere, does it? It displays file parts.


Should the whole of the above comment in fact be talking about which Thor's are listed, not nodes, or parts.
Eclwatch XREF will only show the primary Thor not the other Thor's that share the same node group.